tools/plist_to_html/setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="plist-to-html",
+    version="1.0.0",
+    author='CodeChecker Team (Ericsson)',
+    description="Parse and create HTML files from one or more '.plist' "
+                "result files.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/Ericsson/CodeChecker",
+    keywords=['clang', 'plist-to-html', 'plist', 'html', 'static-analysis',
+              'analysis'],
+    license='LICENSE.txt',
+    packages=setuptools.find_packages(),
+    include_package_data=True,
+    classifiers=[
+        "Environment :: Console",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: University of Illinois/NCSA Open Source License",
+        "Operating System :: OS Independent",
```

Comment:
Was this tested on every OS? I would go with a safe Linux/Unix, OSX until it is not tested on Windows too.